### ISSUE TITLE
File has unexpected coverage changes not found

### ISSUE THREAD

**Issue by damianszczepanik** [codecov/support#164]
New codecov introduces new features but also build failure due to the above error 

https://github.com/damianszczepanik/cucumber-reporting/pull/415

**Comment by Flamefire** [comment]
I have the same problem. I only changed the codecov.yml in a PR and it shows changes in unrelated files. Even worse: The data displayed is wrong. For example: https://codecov.io/gh/Return-To-The-Roots/s25client/src/e51ad1bfb5805575c5ba15eebb9f21bf11c08bf9/src/RTTR_Assert.cpp
It shows a high coverage (>90%) and all lines green. "Toggle messages" and all lines turn to red, which is more correct, as none of these lines got executed. So coverage should be 0%

**Comment by skirpichev** [comment]
Worse, it seems to be impossible to turn off this status check.  According to the docs, something like
```
coverage:
  status:
    changes:
      enabled: false
```
should work, but it doesn't.  I tried several variants in skirpichev/omg#265.

**Comment by adiroiban** [comment]
@skirpichev not sure if codecov.yml is read from master or from branch.

I have also tried various things, inclusing committing `codecove.yml` (wihout dot) on master... but no luck

I think that something is broken here, or needs better documentation :)

**Comment by adiroiban** [comment]
@skirpichev I think that codecov.yml issue is described here #162

**Comment by zbeekman** [comment]
This status failure is the most brain-dead thing I have ever seen. I know I have coverage changes in a file I didn't edit, because I adjusted a unit test. How do we disable this "feature." Honestly, I think this causes much more pain that it resolves... If you can't figure out that you edited a file somewhere and changed your execution path, then you should be banned from programming... This is the biggest anti-feature I have ever seen in a software engineering product. Someone please tell us how to turn it off.

**Comment by zbeekman** [comment]
ping @stevepeak

**Comment by Flamefire** [comment]
@zbeekman Calm down please ;) There is an easy solution:
Disabling the changes check (which is the one, you don't like) can be done by setting `coverage: status: changes: false` in the YML file. (See docu or #186)
This feature might not be useful, if the one who commits also merges the PR. Otherwise it might be useful as this AUTOMATICALLY detects unexpected changes. You can review them and decide if it is ok. Mistakes happen. And this might e.g. help in detecting typos (e.g. inverted if conditions)
If you don't like that they fail your PR, then there is also a setting for that (see docu, something like `set_failure`)

**Comment by zbeekman** [comment]
Thanks @Flamefire . Can you please link me to the documentation you are talking about? The only documentation I can find is on the [wiki](https://github.com/codecov/support/wiki/codecov.yml), which is super sparse, and on the [website](https://codecov.io/docs#configuration), which is also super sparse.

**Comment by Flamefire** [comment]
Yeah, the documentation is a bit hidden: `Interactive documentation can be found on your repository setting page.` (On the website you linked)
--> Open your repo, click on the Settings tab and there on "YAML" (left side) There the YAML file is explained interactively (click on a setting to show its meaning)
I think you want the coverage:status:changes false and exclude it in the layout

**Comment by romanr** [comment]
Also having this nasty problem. Wasted so many hours trying different configs until we found this bug report.
It's year since it's opened and still can only be fixed by workaround by disabling that stage completely.
